test/ResourceManager.test.cpp

```diff
@@ -3,22 +3,20 @@
 #include <stdexcept>
 
 
-TEST(ResourceManager, RefusesAbsolutePaths) {
+TEST(ResourceManager, ConstructResourceManager) {
 	ResourceManager resourceManager("");
 
+	// Refuse absolute paths
 #ifdef _WIN32
 	EXPECT_THROW(resourceManager.GetResourceStream("C:/Archive.vol"), std::runtime_error);
 	EXPECT_THROW(resourceManager.GetResourceStream("C:/PathTo/Archive.vol"), std::runtime_error);
 #endif
 	EXPECT_THROW(resourceManager.GetResourceStream("/Archive.vol"), std::runtime_error);
 	EXPECT_THROW(resourceManager.GetResourceStream("/PathTo/Archive.vol"), std::runtime_error);
-}
 
-TEST(ResourceManager, IgnoreNonFilenamesOnConstruction) {
-	// Attempt to load Resource Manager where directories exist with .vol and .clm 'extensions'
+	// Load Resource Manager where directories exist with .vol and .clm 'extensions'
 	EXPECT_NO_THROW(ResourceManager("./data"));
-}
 
-TEST(ResourceManager, AbortsWithCreationOnFilename) {
+	// Refuse to load when passed a filename
 	EXPECT_THROW(ResourceManager("./data/Directory.clm/.keep"), std::runtime_error);
```

Comment:
I'm maybe a little weary of re-using `.keep` for another purpose here. It's semantically confusing.

----

I'm thinking these tests are maybe a little too different to merge into one. One is about a file or folder name, the other is about the contents of a folder and what happens with later processing.

Reply:
Added an empty.txt file to use instead of .keep to make easier to track. Good point.

We have a couple of empty.txt files now in the different test data directories. Maybe worth looking at consolidating? Maybe nice to keep things separate too though.